map-prototype/index.html

```diff
@@ -0,0 +1,217 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8"/>
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no"/>
+  <title>Map / Philly Mesh</title>
```

Comment:
@garrying Is there a good way to group these deployment-specific parts in a configuration file?